modules/models/src/main/java/org/eclipse/xpanse/modules/models/resource/DeployVariable.java

```diff
@@ -19,28 +18,25 @@
 @Data
 public class DeployVariable {
 
-    @NotNull
     @NotBlank
-    @NotEmpty
     @Schema(description = "The name of the deploy variable")
     private String name;
 
-    @NotNull
-    @NotBlank
-    @NotEmpty
-    @Schema(description = "The description for the deploy variable")
-    private String description;
-
     @NotNull
     @Schema(description = "The kind of the deploy variable")
     private DeployVariableKind kind;
 
-    @NotNull
     @NotBlank
-    @NotEmpty
     @Schema(description = "The type of the deploy variable")
     private String type;
```

Comment:
can we change the type from String to `DeployVariableType`

Reply:
Ok.